explainer.md

```diff
@@ -50,13 +50,13 @@ Aside from the root, an element offered for a transition has the following restr
 - [`contain: paint`](https://developer.mozilla.org/en-US/docs/Web/CSS/contain) which ensures that the element is the containing block for all positioned descendants and generates a stacking context. This implies that the child content will be clipped to the context-box but it can be expanded using ['overflow-clip-margin'](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-clip-margin). Being a stacking context and a containing block allows the element to be treated as a single unit, whereas paint containment simplifies implementation.
 - [`break-inside: avoid`](https://developer.mozilla.org/en-US/docs/Web/CSS/break-inside) which disallows fragmentation ensuring the element content is a single rect, i.e., it doesn't break across lines or columns, again allowing the element to be treated as a single unit.
 
-- [Open question](https://github.com/WICG/shared-element-transitions/issues/71): If the developer tries to break the above rules, what should happen? Is paint containment applied by the browser? Is that element dropped from the transition? Is the whole transition abandoned?
+The developer must add the properties above to an element's style if its offered for a transition. If this constraint is not satisfied at any point during the transition, the transition is aborted. See [issue](https://github.com/WICG/shared-element-transitions/issues/71) for detailed discussion.
```

Comment:
Presumably this would only not be satisfied (or detected as not satisfied) after style resolution

Reply:
It's also unclear to me if we're requiring contain: paint or simply a property which adds paint containment. I'll file a separate issue for this.

Reply:
Yes, we can detect it during style resolution and abort in the same frame.

Reply:
Do you mind saying this? "At any point during transition" sounds too vague

Reply:
Done.